Sources/TSCBasic/Path.swift

```diff
@@ -370,6 +373,10 @@ private struct PathImpl: Hashable {
     /// string consisting of just `.` if there is no directory part (which is
     /// the case if and only if there is no path separator).
     fileprivate var dirname: String {
+#if !os(macOS)
```

Comment:
I would really prefer starting with `#if os(Windows)` here. Such changes can have a pretty bad fallout and I'd rather have us vet these code paths on Windows first (since it's being brought up) so we don't end up regressing well-tested paths.

Reply:
Alright, changed it to `#if os(Windows)`

Reply:
hm, did the new commit not go through? It still says macOS

Reply:
Blah, must have cherry-picked the wrong commit... should be fixed now.